troubleshooting.md

```diff
@@ -107,4 +107,25 @@ lstat /home/myusername/~: no such file or directory
 
   * Replace `~` with `$HOME` or the fully specified directory `/home/myusername`.
     * `$ sudo buildah push alpine oci:${HOME}/myalpine:latest`
+
 ---
+### 5) Rootless podman build fails EPERM on NFS:
+
+NFS enforces file creation on different UIDs on the server side and does not understand User Namespace.
+When a container root process like YUM attempts to create a file owned by a different UID, NFS Server denies the creation.
+See [Shortcomings of Rootless Podman](https://github.com/containers/libpod/blob/master/rootless.md)
+
+#### Symptom
+```console
+$ podman build .
+ERRO[0014] Error while applying layer: ApplyLayer exit status 1 stdout:  stderr: open /root/.bash_logout: permission denied 
+error creating build container: Error committing the finished image: error adding layer with blob "sha256:a02a4930cb5d36f3290eb84f4bfa30668ef2e9fe3a1fb73ec015fc58b9958b17": ApplyLayer exit status 1 stdout:  stderr: open /root/.bash_logout: permission denied
+```
+
+#### Solution
+  * Use `sudo podman`
```

Comment:
This Should be choose one of the following.

1. Setup containers/storage in a different directory not on an NFS Share.
.... 
2.  Otherwise just run podman as root, via sudo podman ...